sw_firewall/README.md

```diff
@@ -1,14 +1,27 @@
 # Software Firewall PoC
 
-The Proof of Concept is made up of three types of containers: traffic sources,
-traffic targets, and proxies.  The initial PoC will be:
+The Proof of Concept is made up of three types of containers:
+
+- traffic sources
+- traffic targets
+- proxies
+
+The initial PoC will be:
```

Comment:
Perhaps Initial PoC should also be marked as title similar to Second PoC with "##"